### PR TITLE
Fix typos to be singular-first person written.

### PR DESCRIPTION
**Pull request by yeyeto2788**
Hello guys!  

Just wanted to make the whole text to be in singular-first person written text. 

For me, it sounds better as in the PR, there only few changes on some phrases.

Regards.

### CODE REVIEW

File: mu/locale/es/LC_MESSAGES/mu.po [hunk 1]
```diff
@@ -23,23 +23,23 @@ msgstr "Registrando en {}"
...
+msgstr "Tiempo de conexión agotado. ¿Es tu ordenador lento o esta ocupado? Libere algunos de los recursos del ordenador y vuelva a intentarlo."
 
 #: mu/debugger/client.py:116
 msgid ""
 "Could not find localhost.\n"
 "Ensure you have '127.0.0.1 localhost' in your /etc/hosts file."
 msgstr ""
 "No se pudo localizar localhost.\n"
-"Asegúrese de tener '127.0.0.1 localhost' en su archivo /etc/hosts."
+"Asegúrate de tener '127.0.0.1 localhost' en su archivo /etc/hosts."
```
Comment (carlosperate) [inline mu/locale/es/LC_MESSAGES/mu.po]:
```suggestion
"Asegúrate de tener '127.0.0.1 localhost' en tu archivo /etc/hosts."
```

File: mu/locale/es/LC_MESSAGES/mu.po [hunk 2]
```diff
@@ -23,23 +23,23 @@ msgstr "Registrando en {}"
...
 
 #: mu/interface/dialogs.py:55
 msgid "Select Mode"
-msgstr "Selección de modo"
+msgstr "Seleccionar de modo"
```
Comment (carlosperate) [inline mu/locale/es/LC_MESSAGES/mu.po]: This one doesn't sound quite right to me, did you mean "Seleccionar modo"?
This is the title for this window:

image

File: mu/locale/es/LC_MESSAGES/mu.po (hunk 1 above)
Comment (yeyeto2788) [reply mu/locale/es/LC_MESSAGES/mu.po]: I didn't see this one :( thanks !

**Comment by yeyeto2788** [comment]
@carlosperate thanks for approving the changes.

I gues that @ntoll is the one to merge the changes, right?

**Comment by ntoll** [comment]
Gracias! :-)

**Comment by yeyeto2788** [comment]
You're welcome. It was a pleasure for me helping. I hope the pages are also merged soon :happy:

**Pull request merged**